<launch>
  <node name="state_machine" pkg="ufs_state_machine" type="state_machine_node" output="screen">
    <rosparam command="load" file="$(find ufs_state_machine)/config/state_machine_params.yaml" />
  </node>
</launch>
